### DISCUSSION TITLE
Speed Insight Pricing

### DISCUSSION THREAD

**Discussion by IVEN21** [vercel/community#6222] [Feature Question]
### Question

Why on a hobby plan can use free speed insight, but on a pro plan speed insight must be add-on to use? I'm on a pro plan but I just need the hobby speed insight.

**Comment by leerob** [comment]
Thank you for the feedback here – as we continue to evolve the pricing of Speed Insights, we will take this into consideration 🙏

**Reply by IVEN21** [reply]
Any update on this?
